Sources/Templates/Extensions/Method+Extension.swift

```diff
@@ -210,8 +217,15 @@ private extension Method {
 
             lines.append("\(accessLevel) var stubbed\(name)Result: \(resultType)")
         }
-        if !isInitializer { // Expectations aren't possible in the initializer
-            lines.append("\(accessLevel) var invoked\(name)Expectation = XCTestExpectation(description: \"\\(#function) expectation\")")
+        if !isInitializer { // Expectations/Confirmations aren't possible in the initializer
+            if annotations.testingFrameworkTypes.contains(.xctest) {
+                lines.append("\(accessLevel) var invoked\(name)Expectation = XCTestExpectation(description: \"\\(#function) expectation\")")
+            }
+
+            if annotations.testingFrameworkTypes.contains(.swiftTesting) {
+                lines.append("\(accessLevel) var invoked\(name)Confirmation: Confirmation?")
+                lines.append("\(accessLevel) var invoked\(name)ConfirmationCount: Int = 1")
```

Comment:
I think we can drop the `: Int`, it's also not there for `varXInvokedCount`

Reply:
I'm not sure about the var name though. As I understand the docs correctly `Confirmation?.confirm(count: x)` will confirm with the given count? Perhaps `\(name)ConfirmationIncrement`?